manage.py

```diff
@@ -13,12 +13,15 @@ def main():
     os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'smallboard.settings')
 
     if 'test' in sys.argv:
-        dotenv_file = os.path.join(BASE_DIR, ".env.test")
+        test_env_files = [".env.test.user", ".env.test"]
```

Comment:
Assuming the file loaded second overrides the settings in the first, I think we want to load `.env.test.user` second? I think of `.env.test` as the defaults, and then users might want to override or add some settings.

Reply:
It isn't overrides, it just loads the first file from that list and then breaks out of the loop

Reply:
Oh, oops, I didn't notice that. I got you now.